wandb/data_types.py

```diff
@@ -1647,8 +1647,15 @@ def __init__(self, box_keys=None, mask_keys=None):
         else:
             mask_keys = _dtypes.ConstType(set(mask_keys))
 
+        if classes is None:
+            classes = _dtypes.UnknownType()
+        elif not isinstance(classes, Classes):
+            raise TypeError("classes must be instance of Classes")
+        else:
+            classes = _ClassesIdType(classes)
```

Comment:
you might need to enhance the classes id type to also support the `name` field - we currently only capture the ID, which is not sufficient enough to know the names